packages/apidom-ls/src/config/openapi/path-item/documentation.ts

```diff
@@ -0,0 +1,23 @@
+
+const documentation = [
+  {
+    // OAS 3.1 adds the last sentence vs OAS3.0.3, re: "rules for resolving"
+    target: '$ref',
+    docs: 'Allows for an external definition of this path item. The referenced structure MUST be in the format of a [Path Item Object](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#pathItemObject). In case a Path Item Object field appears both in the defined object and the referenced object, the behavior is undefined. See the rules for resolving [Relative References](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#relativeReferencesURI).',
```

Comment:
link shoul be to 3.1.0 instead of 3.0.3

Reply:
fixed in [15d7f36](https://github.com/swagger-api/apidom/pull/1946/commits/15d7f36994d6758adcae1c8e576d8a55746e65f3)